.github/workflows/rust-cli-publish.yml

```diff
@@ -110,39 +103,36 @@ jobs:
     steps:
 
     - name: Install Rust toolchain
-      uses: actions-rs/toolchain@v1
-      with:
-        profile: minimal
-        toolchain: stable
-        override: true
+      uses: dtolnay/rust-toolchain@stable
 
-    - uses: actions/download-artifact@v2
+    - uses: actions/download-artifact@v3
       with:
         name: repo
 
     - name: Build
       run: cargo build --verbose --release
 
-    - name: Get Upload URL
-      id: get_upload_url
-      run: echo "::set-output name=upload_url::$(cat release_url.txt)"
-
-    - run: echo ${{ steps.get_upload_url.outputs.upload_url }}
+    - name: Get version
+      id: get-version
+      working-directory: zowex
+      run: echo "ZOWEX_VERSION=$(cargo metadata --no-deps | jq -r .packages[0].version)" >> $GITHUB_OUTPUT
 
     - name: Create Archive
       run: |
         cd target/release
-        gtar -cvzf zowe.tgz zowe
```

Comment:
Previously we had to use `gtar` to work around a bug where `tar` on MacOS included an extra file in the archive: https://github.com/zowe/zowe-cli/pull/1311

This seems to no longer be a problem, so I'm cool with using `tar` 🙂